docs/faq.md

```diff
@@ -192,3 +192,12 @@ In the EarlGrey test target, import the application's app delegate header. In `s
     [delegate resetApplicationForTesting];
 }
 ```
+
+**How do I match on internal classes?**
```

Comment:
Perhaps something like:
How do I create a matcher that matches internal UIKit classes?

is a bit more descriptive?